job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy

```diff
@@ -37,8 +37,8 @@ class Constants {
     static final String SONARCLOUD_FOLDER = 'sonarcloud'
     static final String REPORT_BRANCH = '7.x'
     static final String BUILD_DATE = ''
-    static final String NEXT_PROD_VERSION = '7.13.0'
-    static final String CURRENT_PROD_VERSION = '7.12.0'
+    static final String NEXT_PROD_VERSION = '7.14.0'
+    static final String CURRENT_PROD_VERSION = '7.13.0'
```

Comment:
i believe we need to redefine the constants, but we should probably keep them as it was.
main would be disconnected from prod
7.67.x will be 7.13.x
7.59.x is 7.12.x

Reply:
@Ginxo ?